langserver/workspace_refs.go

```diff
@@ -33,8 +36,23 @@ func (h *LangHandler) handleWorkspaceReference(ctx context.Context, conn JSONRPC
 		pkgPat = h.init.RootImportPath + "/..."
 	}
 
+	var paralellism int
+	e := os.Getenv("WORKSPACE_REFERENCE_PARALLELISM")
```

Comment:
Thanks for fixing this @slimsag :)

I worry about the use of ad hoc environment variables like this, as they affect behavior implicitly and they're easy to overlook. Over time, more might be added and then it becomes difficult to even list all the env variables that can affect the app. Could we move this env variable to a separate package or file (e.g., `config.go`) so they are obviously configuration points and in one place?